staging/src/k8s.io/kubectl/pkg/cmd/create/create_secret.go

```diff
@@ -161,6 +167,7 @@ func NewCmdCreateSecretGeneric(f cmdutil.Factory, ioStreams genericiooptions.IOS
 	cmd.Flags().StringSliceVar(&o.FileSources, "from-file", o.FileSources, "Key files can be specified using their file path, in which case a default name will be given to them, or optionally with a name and file path, in which case the given name will be used.  Specifying a directory will iterate each named file in the directory that is a valid secret key.")
 	cmd.Flags().StringArrayVar(&o.LiteralSources, "from-literal", o.LiteralSources, "Specify a key and literal value to insert in secret (i.e. mykey=somevalue)")
 	cmd.Flags().StringSliceVar(&o.EnvFileSources, "from-env-file", o.EnvFileSources, "Specify the path to a file to read lines of key=val pairs to create a secret.")
+	cmd.Flags().StringArrayVar(&o.EncodedSources, "from-base64-literal", o.EncodedSources, "Specify a key and base64 encoded literal value to insert in secret (i.e. mykey=base64-encoded-value)")
```

Comment:
Adding new flag should be triaged by sig-cli first as it requires strong justification.
Until that time;
/hold